### PR TITLE
Flash worldserver on taskbar on thread start

### PR DESCRIPTION
**Pull request by illunix**
Flush worldserver on taskbar on thread start

**Target branch(es):** 3.3.5/master

- [X] 3.3.5
- [X] master

### CODE REVIEW

**Comment by MaxtorCoder** [comment]
Fix all the checks, please.

**Comment by jackpoz** [comment]
There is a "beep at start" config, this should have a config too. I personally don't want my worldserver to flash.

**Comment by fengzi3209** [comment]
请检查更新后的cmake，现在显示编译错误，用前几天的代码就可以，没有问题，替换源码里的cmake也没有问题。我的系统是win7 64.

**Comment by fengzi3209** [comment]
请检查更新后的cmake，现在显示编译错误，用前几天的代码就可以，没有问题，替换源码里的cmake也没有问题。我的系统是win7 64.

**Comment by fengzi3209** [comment]
抱歉，我不会英文，我来自中国。另外沙塔斯城，奥尔多阵营中立，也无法交付任务物品，显示交战状态。我是部落猎人。

**Comment by MaxtorCoder** [comment]
> 请检查更新后的cmake，现在显示编译错误，用前几天的代码就可以，没有问题，替换源码里的cmake也没有问题。我的系统是win7 64.

English......

File: src/server/worldserver/worldserver.conf.dist [hunk 1]
```diff
@@ -1207,6 +1207,14 @@ Event.Announce = 0
 
 BeepAtStart = 1
 
+#
+#    FlushAtStart
+#        Description: Flush when the world server finished starting.
+#        Default:     1 - (Enabled)
+#                     0 - (Disabled)
+
+FlushAtStart = 1
```
Comment (jackpoz) [inline src/server/worldserver/worldserver.conf.dist]: It should be "flash" here and wherever "flush" is used

Comment (Faq) [reply src/server/worldserver/worldserver.conf.dist]: What is flush? Guessing for consistency should be used same word in this code.

**Comment by Shauren** [comment]
Ugh, enabled by default just like BeepAtStart, personally hate both of them

**Comment by illunix** [comment]
I have the sound turned off while I'm on the pc so something like this will be useful to me and maybe other people

**Comment by illunix** [comment]
Personally I would disable beep on start and have flash only

**Comment by Shauren** [comment]
Can't do that, windows isn't the only os we support

**Comment by illunix** [comment]
> Can't do that, windows isn't the only os we support

So this pr can not be merged?

**Comment by Shauren** [comment]
I meant that as a response to 
> Personally I would disable beep on start and have flash only

**Comment by illunix** [comment]
Ah yeah that make sense, so maybe disable flash by default?

**Comment by jackpoz** [comment]
Looks good to me (maybe the english in the .conf could be improved but that's not a problem)

**Review by jackpoz — APPROVED** [review]

File: src/server/worldserver/worldserver.conf.dist [hunk 2]
```diff
@@ -1207,6 +1207,14 @@ Event.Announce = 0
 
 BeepAtStart = 1
 
+#
+#    FlashAtStart
+#        Description: Flash when the world server finished starting. (Works on Windows only)
```
Comment (Faq) [inline src/server/worldserver/worldserver.conf.dist:1212]:
```suggestion
#        Description: Flashes in taskbar when the world server finished starting. (Works on Windows only)
```

**Comment by jackpoz** [comment]
Thanks for the PR 🚀 😃

**Pull request merged**